packages/middleware-endpoint/src/resolveEndpointConfig.ts

```diff
@@ -1,14 +1,17 @@
 import { Endpoint, EndpointParameters, EndpointV2, Logger, Provider, UrlParser } from "@aws-sdk/types";
 import { normalizeProvider } from "@aws-sdk/util-middleware";
 
+import { toEndpointV1 } from "./adaptors/toEndpointV1";
+
 /**
- * Endpoint config interfaces and resolver for Endpoint v2. They live in separate package to allow per-service onboarding.
- * When all services onboard the endpoint v2, the resolver in config-resolver package can be removed.
+ * Endpoint config interfaces and resolver for Endpoints v2. They live in separate package to allow per-service onboarding.
+ * When all services onboard the Endpoints v2, the resolver in config-resolver package can be removed.
```

Comment:
Retain Endpoint v2? As we use it in other places
```suggestion
 * Endpoint config interfaces and resolver for Endpoint v2. They live in separate package to allow per-service onboarding.
 * When all services onboard the Endpoint v2, the resolver in config-resolver package can be removed.
```

Reply:
I will accept this change, but I think we should in fact rename everything to Endpoints 2.0 or Endpoints V2 (plural), because that is the name of the project.